packages/features/portfolio/src/side-drawers/convert-cash-to-n-tokens/error-msgs.tsx

```diff
@@ -0,0 +1,12 @@
+import { defineMessages } from 'react-intl';
+
+export const errorMsgs = defineMessages({
+  insufficientBalance: {
```

Comment:
You can import the insufficient balance error from here:
https://github.com/notional-finance/notional-monorepo/blob/7206ab2650baec936cfca78fcb1938cef32e0223/packages/shared/trade/src/tradeErrors.tsx#L5